src/component/setting.js

```diff
@@ -100,9 +109,22 @@ const iStyles = StyleSheet.create({
   },
 });
 
-export const SettingItem = ({ name, onSelect, label, arrow, children }) => (
-  <ListItem style={iStyles.item} onSelect={onSelect}>
-    <Text style={iStyles.name}>{name}</Text>
+export const SettingItem = ({
+  name,
+  copy,
+  onSelect,
+  label,
+  arrow,
+  children,
+}) => (
+  <ListItem
+    style={[iStyles.item, copy ? { height: 80 } : null]}
+    onSelect={onSelect}
+  >
+    <View style={iStyles.left}>
+      <Text style={iStyles.name}>{name}</Text>
+      {copy ? <Text style={iStyles.copy}>{copy}</Text> : null}
+    </View>
```

Comment:
Would probably be better to create a separate `SettingCopyItem` component than trying to squeeze both use cases into one component. That would also allow us to set the margins/padding according to the design, as they look pretty different right now.

Design:
<img width=200 src="https://user-images.githubusercontent.com/1374174/63275839-89914400-c2a2-11e9-879d-de53276b8b89.png" />

Implementation:
<img width=200 src="https://user-images.githubusercontent.com/1374174/63275863-93b34280-c2a2-11e9-9596-5f220027a986.png" />

Reply:
Thanks, must've been in a hurry hah